### PR TITLE
Add footer

### LINKED ISSUE #6: Footer
#### Screenshots
Screenshot 2020-03-14 at 23 15 38

-----

#### Used by
 - All the apps

#### Names
 - Footer

#### Children
 - Links

### PR DESCRIPTION
**Pull request by DanielStoica85**
Closes #6 

Added Footer component.

### CODE REVIEW

**Comment by RaduCStefanescu** [comment]
@DanielStoica85 could you please resolve the merge conflict?

**Comment by DanielStoica85** [comment]
> @DanielStoica85 could you please resolve the merge conflict?

Trying to figure out how to do that.

File: src/components/footer/footer.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <div className="code4-branding">
+          <a
+            href="https://code4.ro/ro/"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            {/* <img src={Code4romania_gray} alt="Code for Romania gray logo"></img> */}
+            <Code4RomaniaGrayLogo style={{ height: 40 }} />
+          </a>
+          <p>© 2020 Code for Romania.</p>
```
Comment (surdu) [inline src/components/footer/footer.js]: It would be a good idea to make the current year dynamic here

Comment (DanielStoica85) [reply src/components/footer/footer.js]: Good idea! Done.

**Comment by surdu** [comment]
Awesome work! Green from me!

**Pull request merged**